libs/ui/src/components/atoms/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from './dropdownStyle';
+import React from 'react';
+import { v4 as uuid } from 'uuid';
+import { useTheme } from 'styled-components';
+import { Theme } from '../../../types/theming';
+import { DropdownLabel } from './dropdownExtras';
+
+const DropdownContentOptions = {
+  clickable: DropdownMenuItem,
+  label: DropdownLabel,
+};
+
+export type DropdownItem = {
+  type: keyof typeof DropdownContentOptions;
+  content: React.ReactNode;
+};
+
+type DropdownProps = {
+  trigger: React.ReactNode;
+  items: DropdownItem[];
+  bg: string;
+  spacing: string;
+};
+// TODO aria
+const Dropdown = ({ trigger, items, bg, spacing = '0' }: DropdownProps) => {
+  const theme = useTheme() as Theme;
```

Comment:
Agreed, but I can't really find an alternative.

Figured it would be safe to cast because the context is synchronously initialized and set with theme-shaped objects. Even if I explicitly pass in the Theme type to the state, for some reason it's still trying to use DefaultState.

Reply:
This might be the way to override default theme https://github.com/styled-components/styled-components/blob/15dbc4e46e5954820fbd48b40842cd31d821c855/packages/styled-components/src/models/ThemeProvider.tsx#L6

Reply:
That worked